Server/src/RedisManager.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2024 Microsoft, Inc. and others.  All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ */
+
+#ifndef REDISMANAGER_H_
+#define REDISMANAGER_H_
+
+#include <sonic-swss-common/common/dbconnector.h>
+#include <sonic-swss-common/common/table.h>
+#include <sonic-swss-common/common/select.h>
+#include <sonic-swss-common/common/selectableevent.h>
+#include <sonic-swss-common/common/subscriberstatetable.h>
+
+#include <string>
+#include <list>
+#include <map>
+#include <unordered_set>
+#include <functional>
+#include <vector>
+#include "Logger.h"
+#include "BMPListener.h"
+
+
+/**
+ * BMP_TABLE_* defines the default table name prefix
+ */
+#define BMP_DB_NAME                "STATE_DB"
+#define BMP_TABLE_NEI              "BGP_NEIGHBOR_TABLE"
+#define BMP_TABLE_RIB_IN           "BGP_RIB_IN_TABLE"
+#define BMP_TABLE_RIB_OUT          "BGP_RIB_OUT_TABLE"
+#define BMP_TABLE_NEI_PREFIX       "BGP_NEIGHBOR"
+#define BMP_TABLE_NEI_KEYS         "KEYS BGP_NEIGHBOR*"
```

Comment:
>BMP_TABLE_NEI_KEYS [](http://example.com/codeflow?start=8&length=18)

It is more like  "KEYS "+BMP_TABLE_NEI_PREFIX+"*".

Reply:
removed.